assets/docs/rishit-dagli/plant-disease/default/tfjs/1.md

```diff
@@ -0,0 +1,30 @@
+# Tfjs rishit-dagli/plant-disease/default/tfjs/1
```

Comment:
If you want to simplify the final serving URL a little, you could change this to:

`# Tfjs rishit-dagli/plant-disease/default/1`. That would change the final URL to be https://tfhub.dev/rishit-dagli/tfjs-model/plant-disease/default/1. You do *not* need to change the actual location of this markdown file to make this change.

Reply:
Sure thing, I will also make this edit to the TF Lite model